Brocfile.js

```diff
@@ -33,20 +33,26 @@ var es5 = new Babel(lib, {
   blacklist: ['es6.modules']
 });
 
+function rollupConfig(entry) {
+  return new Rollup(es5, {
+    rollup: {
+      entry: 'lib/' + entry,
+      cache: 'BUILD:es6-promise',
```

Comment:
https://github.com/chadhietala/broccoli-rollup/commit/5aaf8f38f723a3d6e0f6160fa65888ed65c0329d
That was added by you. I feel like such an idiot.